### ISSUE TITLE
Serializer for WorldGuard regions not found

### ISSUE THREAD

**Issue by TPGamesNL** [SkriptLang/Skript#3203]
### Description
`WorldGuardRegion`s can't be serialized even though it should be able to (`WorldGuardHook.WorldGuardRegion` extends `Region` which extends `YggdrasilExtendedSerializable`). It looks like `Yggdrasil#getSerializer(Class)` returns `null`.

### Steps to Reproduce
Use
```
on region enter:
	set {region} to event-region
```
and enter a region.

### Expected Behavior
No error and the variable is saved normally

### Errors / Screenshots
https://pastebin.com/YRPWv6wj

### Server Information
* **Server version/platform:** 1.15.2 (paper-379)
* **Skript version:** 2.5-alpha6(rev1)

### Additional Context
https://github.com/SkriptLang/Skript/issues/3185 could be related / have the same underlying issue

**Comment by ShaneBeee** [comment]
I believe this issue was related the #3185 
Can you please retest and let me know what happens. 
Hopefully by resolving that underlying issue it will resolve this one as well.

**Comment by TPGamesNL** [comment]
Issue is still present in 2.5-beta1. About #3185, which commit fixed it? I can only find the commit that disabled the feature that caused the issue (https://github.com/SkriptLang/Skript/commit/50ae1d500cf449eac20aa8e31c07373593c5a5eb#diff-fa072974f44e81d7b12a1332af9f1f95), not the one that fixed it

**Comment by ShaneBeee** [comment]
Yeah nothing actually "fixed" that issue, we just disabled the stuff that caused that issue.

Im not able to replicate the issue. Are you by any chance running FAWE's WorldGuard?

**Comment by TPGamesNL** [comment]
I'm running paper 385 (1.15.2), worldedit 7.1.0, worldguard 7.0.3, Skript 2.5-beta1

**Comment by ShaneBeee** [comment]
Ok I CAN replicate this issue.

**Comment by Wafffidooe** [comment]
Isn't the "waiting for reply" label supposed to be removed? :)
